service/src/main/java/com/commercetools/pspadapter/payone/mapping/PayoneRequestFactory.java

```diff
@@ -37,8 +52,7 @@ public CaptureRequest createCaptureRequest(final PaymentWithCartLike paymentWith
     }
 
     protected void mapFormPaymentWithCartLike(final AuthorizationRequest request,
-                                              final PaymentWithCartLike paymentWithCartLike,
-                                              final Logger logger) {
+                                              final PaymentWithCartLike paymentWithCartLike) {
```

Comment:
some methods use use `@Nonnull` and `@Nullable` annotations and some you don't use at all, Is there a reason? Isn't it better to consistently use them?